.github/workflows/ci.yaml

```diff
@@ -17,7 +17,8 @@ jobs:
         run: |
           go get ./pinecone
       - name: Run tests
-        run: go test ./pinecone
+        #        run: go test -count=1 -v ./pinecone
```

Comment:
We will re-enable this (and remove the line below) once the tests for `client` are working

Reply:
Yeah -- lemme rebase your changes (where you added the new env vars for the `client` integration tests) into mine once you merge, and then it'll be done